app/views/config_reports/_list.html.erb

```diff
@@ -27,12 +27,12 @@
         <% end %>
         <td><%= reported_at_column(report) %></td>
         <td class="fact-origin"><%= report_origin_icon(report.origin) %></td>
-        <td><%= report_event_column(report.applied, "label-info") %></td>
-        <td><%= report_event_column(report.restarted, "label-info") %></td>
-        <td><%= report_event_column(report.failed, "label-danger") %></td>
-        <td><%= report_event_column(report.failed_restarts, "label-warning") %></td>
-        <td><%= report_event_column(report.skipped, "label-info") %></td>
-        <td><%= report_event_column(report.pending, "label-info") %></td>
+        <td><%= report_event_column(report.applied_s, "label-info") %></td>
```

Comment:
Aren't we using method, that you're deprecating here?
```suggestion
        <td><%= report_event_column(report.calculator.status_as_text_of('applied'), "label-info") %></td>
```
or define the methods as I'm suggesting ^^

Reply:
I think this was a copy and paste error. I wanted to deprecate `applied` method and force to use  `applied_s` method instead. I still think that delegation is okay, makes sense.